examples/app-dir-mdx/app/page.tsx

```diff
@@ -60,7 +60,7 @@ export default function Home() {
           <h2>
             Templates <span>-&gt;</span>
           </h2>
-          <p>Explore the Next.js 13 playground.</p>
+          <p>Explore starter templates for Next.js.</p>
```

Comment:
Do we also need to update these in the `create-next-app` templates?

Reply:
It has been updated in #57532